org.kde.cervisia.json

```diff
@@ -0,0 +1,61 @@
+{
+  "app-id": "org.kde.cervisia",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-22.08",
+  "sdk": "org.kde.Sdk",
+  "command": "cervisia",
+  "rename-icon": "cervisia",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--talk-name=org.kde.klauncher5",
+    "--talk-name=org.kde.cvsservice5",
+    "--system-talk-name=org.freedesktop.UDisks2"
+  ],
+  "modules": [
+    {
+      "name": "kdesu",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.kde.org/stable/frameworks/5.108/kdesu-5.108.0.tar.xz",
+          "sha256": "d08b1ccd280def0550596c8d2e3ca35167536fa3826e676bfe290193c37d1ee6",
+          "x-checker-data": {
+            "type": "anitya",
+            "project-id": 8762,
+            "stable-only": true,
+            "url-template": "https://download.kde.org/stable/frameworks/$major.$minor/kdesu-$version.tar.xz"
+          }
+        }
+      ]
+    },
+    {
+      "name": "cervisia",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DBUILD_TESTING=OFF"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.kde.org/stable/release-service/22.12.3/src/cervisia-22.12.3.tar.xz",
```

Comment:
23.04.3

Reply:
But besides that, the fact that the app doesn't start is obviously a bigger problem ^_^